src/components/OptionsSelector/BaseOptionsSelector.js

```diff
@@ -189,8 +207,47 @@ class BaseOptionsSelector extends Component {
         return defaultIndex;
     }
 
+    /**
+     * When pagination is enabled,
+     * maps sections to render only allowed count of them.
```

Comment:
If I understand correctly, this means that the data is split into "sections" of 500. And this function gets the data for a specific section. Is that right?

Reply:
Ah ok, I get it now. Thanks!